### DISCUSSION TITLE
Pass additional data when creating a new widget

### DISCUSSION THREAD

**Discussion by unidesigner** [caplin/FlexLayout#333] [Q&A] [answered]
I'd like to use the `addTabToActiveTabSet ` function and add additional data fields that I can then use in my `const factory = (node) => {` function to pass on in the component creation. Is this already possible? Thanks.

**Comment by nealus** [comment, marked as answer]
you can pass data in the config attribute within the json

**Reply by unidesigner** [reply]
thanks, great - that worked with `node.getConfig()`
